### DISCUSSION TITLE
Template Error: IAM "x" doesn't exist

### DISCUSSION THREAD

**Discussion by williamsullivan** [mozilla/hubs#4580] [Hubs Cloud Questions]
Following the steps on a new hubs cloud enterprise dev server, but we are getting a ROLLBACK_COMPLETE error. 

The first error in the event list:

NearsparkApplication | CREATE_FAILED | Template error: IAM role HubsCloudEngine-Nearspark doesn't exist

and then a slew of other errors:

ToLower | CREATE_FAILED | Template error: IAM role HubsCloudEngine-ToLowerRole doesn't exist

I have tried to remove the root stack several times. cancelled the marketplace subscription, reactivated, and then still keep getting these same errors. 
![Capture](https://user-images.githubusercontent.com/1209421/130994039-e77decca-ee56-459b-97ca-1169d0d37183.JPG)

**Comment by pattersonbl2** [comment]
Are you running any custom setting?   

Does your account have proper permissions to create IAM roles ?
